### PR TITLE
fix/remove delete option

### PR DESCRIPTION
**Pull request by onfranciis**
## Description
Remove the delete option in the recent table drop down of the dashboard

## Related Issues
- #474 

## Steps to Test
- Go to 'dashboard'
- Scroll down to recent section
- Click any of the row's option

## Screenshots (if applicable)
Screenshot from 2023-09-07 00-26-58

## Checklist

- [x] I have tested these changes
- [x] I have updated the relevant documentation
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the codebase
- [x] My changes generate no new warnings or errors
- [x] The title of my pull request is clear and descriptive

### CODE REVIEW

**Comment by onfranciis** [comment]
Just made an update; I added the download option in the drop down. For this feature to be tested correctly, PR #473 should have been approved and merged.
Screenshot from 2023-09-07 01-46-09

File: frontend/src/modules/DashboardModule/components/RecentTable/index.jsx [hunk 1]
```diff
@@ -15,7 +22,9 @@ function DropDownRowMenu({ row, entity }) {
   function Edit() {
     history.push(`/${entity}/update/${row._id}`);
   }
-  function Delete() {}
+  function Delete() {
+    window.open(`${DOWNLOAD_BASE_URL}${entity}/${row._id}`, '_blank');
+  }
```
Comment (salahlalami) [inline frontend/src/modules/DashboardModule/components/RecentTable/index.jsx:27]: change name of function to download

**Review by salahlalami — CHANGES_REQUESTED** [review]

File: frontend/src/modules/DashboardModule/components/RecentTable/index.jsx [hunk 2]
```diff
@@ -15,7 +22,9 @@ function DropDownRowMenu({ row, entity }) {
   function Edit() {
     history.push(`/${entity}/update/${row._id}`);
   }
-  function Delete() {}
+  function Download() {
+    window.open(`${DOWNLOAD_BASE_URL}${entity}/${row._id}`, '_blank');
+  }
 
   return (
     <Menu style={{ width: 130 }}>
```
Comment (polymahh) [inline frontend/src/modules/DashboardModule/components/RecentTable/index.jsx:30]: can we change width to 140
or just say Download instead of Download PDF

**Review by salahlalami — APPROVED** [review]

File: frontend/src/modules/DashboardModule/components/RecentTable/index.jsx (hunk 2 above)
Comment (onfranciis) [reply frontend/src/modules/DashboardModule/components/RecentTable/index.jsx:30]: Alright

**Review by polymahh — APPROVED** [review]

**Pull request merged**